test/persistence-hooks.test.js

```diff
@@ -15,4 +15,6 @@ for(var i in config) {
   customConfig[i] = config[i];
 }
 
-suite(global.getDataSource(customConfig), should);
+suite(global.getDataSource(customConfig), should,
+  {replaceOrCreateReportsNewInstance: (process.env.MONGODB_VERSION &&
+  !semver.satisfies(process.env.MONGODB_VERSION, '>=2.6.0'))});
```

Comment:
This is too terse to my taste. Could you please refactor so that we can end up with something like the following?

```js
suite(global.getDataSource(customConfig), should, {
  replaceOrCreateReportsNewInstance: /* a short statement */
});
```

For example:

```js
var DB_VERSION = process.env.MONGODB_VERSION;
var DB_HAS_2_6_FEATURES = semver.satisfies(DB_VERSION, '>=2.6.0');

suite(global.getDataSource(customConfig), should, {
  replaceOrCreateReportsNewInstance: DB_HAS_2_6_FEATURES
});
```